Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) 2021 Red Hat, Inc.
 # Copyright Contributors to the Open Cluster Management project
 # Build the manager binary
-FROM registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux AS builder
+FROM registry.ci.openshift.org/stolostron/builder:go1.16-linux AS builder
```

Comment:
Should we update to go1.17 too?

Reply:
I have those changes in https://github.com/stolostron/search-operator/pull/94

Reply:
Thanks, approved #94.